File: level22/homework/hw3.py
# 3) მომხმარებელს შემოატანინეთ რიცხვი შემდეგ კი ერთიდან ამ რიცხვამდე დაბეჭდეთ ყველა რიცხვის კვადრატის ჯამი


ran = int(input("enter your number: "))

sum = 0

for i in range(ran):
    sum += (i ** 2)
    

print(sum)   

File: level100/homework/hw3.js
// 3)მომხმარებელს prompt–ით შეეკითხე რიცხვი.

// თუ რიცხვი > 0 → დაბეჭდე "დადებითია".

// თუ რიცხვი < 0 → დაბეჭდე "უარყოფითია".

// თუ 0 → დაბეჭდე "ნულია".

let number = Number(prompt("enter number"));

if (number > 0) {
  console.log("დადებითია");
} else if (number < 0) {
  console.log("უარყოფითია");
} else {
  console.log("ნულია");
}
